src/datalinker/handlers/external.py

```diff
@@ -105,6 +106,7 @@ async def get_index(
     ``metadata`` that provides the same Safir-generated metadata as the
     internal root endpoint.
     """
+    config = config_dependency.config()
```

Comment:
It's a dependency, so it should be a method argument using FastAPI's `Depends`, here and below.